ddtrace/http/headers.py

```diff
@@ -0,0 +1,106 @@
+import re
+
+
+REQUEST = 'request'
+RESPONSE = 'response'
+
+
+def store_request_headers(headers, span, white_list):
+    """
+    Store request headers as a span's tags
+    :param headers: All the request's http headers, will be filtered through the whitelist
+    :dict headers: dict
+    :param span: The Span instance where tags will be store
```

Comment:
~store~ stored

Reply:
You mean `stores`? I meant the present of the verb.

Reply:
In this case it should be "stored"